notebooks/tm/landcover/01-extract-landcover-features.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<ol><li>Change column name format to  <code>landcover_pct_{col}</code></li><li>Keep only pct_ columns (remove total area columns)</li><li>Fill all NaN with 0</li><li>Please limit results to 2 decimal places</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>